README.md

```diff
@@ -1,70 +1,52 @@
-# Getting Started with Create React App
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-## Available Scripts
+# LeaderBoard
 
-In the project directory, you can run:
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-### `npm start`
+"Math magicians" is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to Make simple calculations, Read a random math-related quote built with ReactJS and scss
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## 🧰 Built With
 
-### `npm test`
+- HTML
+- JS
+- CSS
+- JSX
+- ReactJS
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## Live Demo
 
-### `npm run build`
+Not Available Yet.
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+## Getting Started
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+### To launch the project locally:
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+#### Step 1:
+- Clone this repo locally using git and the following git command
 
-### `npm run eject`
+  git clone https://github.com/USERNAME/REPOSITORY
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+#### Step 2:
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+- Run index.html file in your browser
```

Comment:
[ OPTIONAL CHANGE ]

To start a react app, I believe after accessing the correct path of the cloned project, one needs to enter `npm start` and not the index.html file because the app runs on a server. Please update that in your README.